src/content/docs/en/glossary.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Glossary 
+description: A collection of terms and definitions used in the Astro documentation and community.
+i18nReady: false
+---
+
+This glossary is intended to explain the terminology used in both the Astro documentation and in the Astro community.
```

Comment:
```suggestion
This glossary describes common terminologies used in both the Astro documentation and in the Astro community.
```

Reply:
```suggestion
This glossary is intended to describe common terminology used in the Astro documentation and the Astro community.
```

@Genteure how is this?

Reply:
```suggestion
This glossary describes common terminology used in the Astro documentation and the Astro community.
```

This is also good, in my opinion.

Reply:
You can choose one you prefer! Both are good to me, I don't really have a strong opinion on either one. Maybe the first one you suggested?